packages/terra-worklist-data-grid/tests/wdio/worklist-data-grid-spec.js

```diff
@@ -28,98 +28,95 @@ Terra.describeViewports('WorklistDataGrid', ['medium', 'large'], () => {
     });
 
     it('up arrow key cannot navigate outside grid', () => {
       browser.keys(['Tab']
-        .concat(new Array(1).fill('ArrowDown'))
         .concat(new Array(3).fill('ArrowUp')));
 
+      browser.pause(150);
       Terra.validates.element('first-cell-in-grid-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('left arrow key cannot navigate outside grid', () => {
       browser.keys(['Tab']
-        .concat(new Array(1).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowUp'))
-        .concat(new Array(3).fill('ArrowLeft')));
+        .concat(new Array(2).fill('ArrowLeft')));
 
+      browser.pause(150);
       Terra.validates.element('first-cell-in-grid-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('control with home key', () => {
       browser.keys(['Tab']
-        .concat(new Array(4).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowRight'))
+        .concat(new Array(2).fill('ArrowDown'))
+        .concat(new Array(2).fill('ArrowRight'))
         .concat(['Control', 'Home', 'Control']));
 
+      browser.pause(150);
       Terra.validates.element('first-non-column-header-cell-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('home key', () => {
       browser.keys(['Tab']
-        .concat(new Array(4).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowRight'))
+        .concat(new Array(2).fill('ArrowDown'))
+        .concat(new Array(2).fill('ArrowRight'))
         .concat(['Home']));
 
+      browser.pause(150);
       Terra.validates.element('first-cell-in-row-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('end key', () => {
       browser.keys(['Tab']
-        .concat(new Array(4).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowRight'))
+        .concat(new Array(1).fill('ArrowDown'))
         .concat(['End']));
 
+      browser.pause(150);
       Terra.validates.element('last-cell-in-row-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('control with end key', () => {
       browser.keys(['Tab']
-        .concat(new Array(4).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowRight'))
         .concat(['Control', 'End', 'Control']));
 
+      browser.pause(150);
       Terra.validates.element('last-cell-in-last-row-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('down arrow cannot navigate outside grid', () => {
       browser.keys(['Tab']
         .concat(['Control', 'End', 'Control'])
-        .concat(new Array(3).fill('ArrowDown')));
+        .concat(new Array(2).fill('ArrowDown')));
```

Comment:
This seems overkill for an array of size 2
```suggestion
        .concat(['ArrowDown', ArrowDown']);
```

Reply:
This seems to be more of a programmer preference for the spec test, right?  It does not impact performance or readability from what I can see.